src/pages/faq/comparison/ampt.mdx

```diff
@@ -0,0 +1,56 @@
+export const description = 'Comparison to Ampt'
+
+# Ampt vs. Nitric
+
+Nitric is a framework that empowers developers to build complete cloud applications on their preferred cloud platform, combining infrastructure declaration and application code in one place. On the other hand, Ampt is a development platform tailored for building cloud backend applications using the JavaScript/TypeScript language, with deployment options only for AWS. Both serve the purpose of simplifying cloud application development but operate in slightly different ways and cater to different specific needs.
+These are the main differences between the two:
+
+1. **Cloud Provider Support:** Nitric is provider-agnostic, capable of deploying to multiple cloud providers such as AWS, Google Cloud, and Azure. Ampt is primarily designed for AWS services.
+2. **Language Support:** Nitric provides multiple libraries for TypeScript/JavaScript, Python, Go, C# .NET and Java, allowing developers to choose their preferred language. Currently Ampt only supports TypeScript and JavaScript.
+3. **Infrastructure Provisioning:** Nitric uses Pulumi by default for provisioning cloud resources, but also allows the use of [custom providers](/reference/providers/custom/building-custom-provider). Ampt uses a hosted provisioning engine via their online platform.
+4. **Local Simulation:** Nitric provides a fully offline local cloud simulation tool, allowing you to test your applications locally before deploying them. This is a feature that Ampt does not offer, for local development Ampt deploys and updates your application to the cloud via a sandbox environment.
+5. **Open Source:** As an open-source framework, Nitric does not require a platform or subscription to deploy applications.
```

Comment:
Would it be beneficial to add a comment about Ampt being closed source? Currently its implied, not stated explicitly.

Reply:
Changed